packages/api/src/modules/notify/outPipes/BrevoContactNotifyPipe.ts

```diff
@@ -78,11 +78,15 @@ export class BrevoContactNotifyPipe extends BrevoNotifyPipe implements NotifyOut
                     // It's error code for { code: 'duplicate_parameter', message: 'Contact already exist' }
                     // If user exists in other list brevo throws an error so we update contact to add in good list
                     return sendUpdate()
-                        .then(({ response }) => {
-                            const status = response.statusCode || 0;
+                        .then(updateAnwser => {
+                            if (!updateAnwser.response) {
+                                Sentry.captureException(updateAnwser);
+                                return false;
+                            }
+                            const status = updateAnwser.response.statusCode || 0;
                             if (status > 200 && status < 300) return true;
```

Comment:
"200" c'est bon non ? 
```suggestion
                            if (status >= 200 && status < 300) return true;
```

Reply:
Humm, j'en sais rien ;D

Reply:
Théoriquement oui, en pratique je sais pas trop

Reply:
Plutôt si quand même, ce serait trop bizarre

Reply:
Pour moi soit on met `>=` soit on cible direct 204 comme prévoit [la doc](https://developers.brevo.com/reference/updatecontact)